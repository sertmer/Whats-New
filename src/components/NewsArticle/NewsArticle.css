img {
  height: 300px;
  width: 500px;
  padding: 10px;
}

.article {
  margin: 2%;
  width: 45%;
  background-color: #ffffff;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
}

.link-div {
  background-color: #7d48a8;
  font-size: x-large;
  color: #ffffff;
  padding: 1%;
}

.article-title {
  font-size: x-large;
  color: black;
}