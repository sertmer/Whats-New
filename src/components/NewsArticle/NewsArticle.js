import React from 'react';
import './NewsArticle.css';

const NewsArticle = (props) => {
  return (
    <article className='article'>
      <div>
        <img src={props.img}/>
      </div>
      <h1 className='article-title'>{props.headline}</h1>
      <p className='article-description'>{props.description}</p>
      <div className='link-div'>
        <a href={props.url}>Link to Article</a>
      </div>
    </article>
  )
  }

export default NewsArticle;