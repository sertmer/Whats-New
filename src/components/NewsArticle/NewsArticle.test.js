import React from 'react'
import NewsArticle from './NewsArticle'
import { shallow } from 'enzyme'

describe('NewsArticle', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<NewsArticle 
      headline={'b'}
      description={'b'}
      img={'c'}
      url={'d'}
      /> )

    expect(wrapper).toMatchSnapshot()
  })
})