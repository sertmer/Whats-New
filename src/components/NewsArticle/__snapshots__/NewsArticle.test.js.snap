// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticle should match the snapshot 1`] = `
<article
  className="article"
>
  <div>
    <img
      alt=""
      src="c"
    />
  </div>
  <h1
    className="article-title"
  >
    b
  </h1>
  <p
    className="article-description"
  >
    b
  </p>
  <div
    className="link-div"
  >
    <a
      href="d"
    >
      Link to Article
    </a>
  </div>
</article>
`;
